src/sonic-config-engine/tests/sample-arista-7260-t0-minigraph.xml

```diff
@@ -0,0 +1,2869 @@
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet61/3</EndPort>
+        <StartDevice>Servers105</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet62/1</EndPort>
+        <StartDevice>Servers106</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet62/3</EndPort>
+        <StartDevice>Servers107</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet63/1</EndPort>
+        <StartDevice>Servers108</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet63/3</EndPort>
+        <StartDevice>Servers109</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet64/1</EndPort>
+        <StartDevice>Servers110</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>str-7260cx3-acs-2</EndDevice>
+        <EndPort>Ethernet64/3</EndPort>
+        <StartDevice>Servers111</StartDevice>
+        <StartPort>eth0</StartPort>
+      </DeviceLinkBase>
+    </DeviceInterfaceLinks>
+    <Devices>
+      <Device i:type="ToRRouter">
+        <Hostname>str-7260cx3-acs-2</Hostname>
+        <HwSku>Arista-7260CX3-D108C8</HwSku>
+        <ManagementAddress xmlns:a="Microsoft.Search.Autopilot.NetMux">
+           <a:IPPrefix>10.64.247.10</a:IPPrefix>
+        </ManagementAddress>
+      </Device>
+      <Device i:type="LeafRouter">
+         <Hostname>ARISTA04T1</Hostname>
+         <ManagementAddress xmlns:a="Microsoft.Search.Autopilot.NetMux">
+           <a:IPPrefix>10.64.247.203</a:IPPrefix>
+         </ManagementAddress>
+         <HwSku>Arista-VM</HwSku>
+      </Device>
+      <Device i:type="EdgeZoneAggregator">
+         <Hostname>ARISTA03T1</Hostname>
+         <ManagementAddress xmlns:a="Microsoft.Search.Autopilot.NetMux">
+           <a:IPPrefix>10.64.247.202</a:IPPrefix>
+         </ManagementAddress>
+         <HwSku>Arista-VM</HwSku>
+      </Device>
+      <Device i:type="LeafRouter">
```

Comment:
Make all the neighbors of type EdgeZoneAggregator

Reply:
Ok, but this has no effect on cable length since the rest of the neighbors are all leaf routers.

Reply:
This is just to mimic actual environment where all neighbors are EdgeZone devices